scripts/rpc/nvmf.py

```diff
@@ -158,6 +159,8 @@ def nvmf_create_transport(client,
         params['dif_insert_or_strip'] = dif_insert_or_strip
     if sock_priority:
         params['sock_priority'] = sock_priority
+    if wr_batching:
```

Comment:
This will not work when you want to send False. It should be 'if wr_batching is not None:'
See, how it's done for initiator batching